centreon/tests/e2e/features/Cloud-notifications/common.ts

```diff
@@ -180,11 +215,342 @@ const waitUntilLogFileChange = (): Cypress.Chainable => {
+      check_attempt,
+      check_command,
+      check_freshness,
+      check_interval,
+      check_period,
+      check_type,
+      checked,
+      default_active_checks,
+      default_event_handler_enabled,
+      default_flap_detection,
+      default_notify,
+      default_passive_checks,
+      display_name,
+      enabled,
+      event_handler,
+      event_handler_enabled,
+      execution_time,
+      first_notification_delay,
+      flap_detection,
+      flap_detection_on_critical,
+      flap_detection_on_ok,
+      flap_detection_on_unknown,
+      flap_detection_on_warning,
+      flapping,
+      freshness_threshold,
+      high_flap_threshold,
+      icon_image,
+      icon_image_alt,
+      last_hard_state,
+      last_update,
+      latency,
+      low_flap_threshold,
+      max_check_attempts,
+      next_check,
+      no_more_notifications,
+      notification_interval,
+      notification_number,
+      notification_period,
+      notify,
+      notify_on_critical,
+      notify_on_downtime,
+      notify_on_flapping,
+      notify_on_recovery,
+      notify_on_unknown,
+      notify_on_warning,
+      obsess_over_service,
+      output,
+      passive_checks,
+      percent_state_change,
+      perfdata,
+      retain_nonstatus_information,
+      retain_status_information,
+      retry_interval,
+      scheduled_downtime_depth,
+      should_be_scheduled,
+      stalk_on_critical,
+      stalk_on_ok,
+      stalk_on_unknown,
+      stalk_on_warning,
+      state,
+      state_type,
+      volatile
+    ].join('\t');
+
+    centreonStorageServicesValues += values + '\n';
+  }
+  cy.writeFile(
+    './fixtures/notifications/centreon_storage_services.txt',
+    centreonStorageServicesValues
+  );
+  cy.log('Values generated and stored in centreon_storage_services.txt.');
+
+  // Generate values for centreon_services.txt
+  let centreonServicesValues = '';
+  // The first service will got an id of 28
+  for (let i = 28; i < 1028; i++) {
+    const service_id = i;
+    const service_template_model_stm_id = 3;
+    const service_description = `service_${i - 27}`;
+    const service_is_volatile = 2;
+    const service_max_check_attempts = 1;
+    const service_active_checks_enabled = 0;
+    const service_passive_checks_enabled = 1;
+    const service_parallelize_check = 2;
+    const service_obsess_over_service = 2;
+    const service_check_freshness = 2;
+    const service_event_handler_enabled = 2;
+    const service_flap_detection_enabled = 2;
+    const service_process_perf_data = 2;
+    const service_retain_status_information = 2;
+    const service_retain_nonstatus_information = 2;
+    const service_notifications_enabled = 2;
+    const contact_additive_inheritance = 0;
+    const cg_additive_inheritance = 0;
+    const service_inherit_contacts_from_host = 1;
+    const service_use_only_contacts_from_host = 0;
+    const service_locked = 0;
+    const service_register = 1;
+    const service_activate = 1;
```

Comment:
you don't need to declare all those constants if you only use those in `values` array

Reply:
Declaring those constants makes the code more comprehensive; otherwise, nobody would know which column each value refers to.